src/Microsoft.ML.CodeGenerator/Microsoft.ML.CodeGenerator.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <None Update="Templates\Console\Annotation.ttinclude">
       <Generator>TextTemplatingFileGenerator</Generator>
```

Comment:
We should remove the `Generator` and `LastGenOutput` from this. That will fix it so it doesn't re-gen the Annotations.cs file.